File: 프로그래머스/2/42586. 기능개발/기능개발.js
function solution(progresses, speeds) {
    const diploy = progresses.map((el, idx) => Math.ceil((100 - el) / speeds[idx]))
    
    const result = [];
    let count = 0;
    let temp = diploy[0];
    for(let i = 0; i < diploy.length; i++){
        if(temp < diploy[i]){
            result.push(count);
            temp = diploy[i]
            count = 1;
        }else{
            count++
        }
    }
    if(count > 0) result.push(count)
    return result
}

File: 프로그래머스/2/42586. 기능개발/기능개발.py
import math

def solution(progresses, speeds):
    diploy = [math.ceil((100 - progresses[i]) / speeds[i]) for i in range(len(progresses))]
    
    result = [];
    count = 0;
    temp = diploy[0];
    for i in range(len(diploy)):
        if(temp < diploy[i]):
            result.append(count);
            temp = diploy[i];
            count = 1;
        else: count += 1
    if count > 0: result.append(count)
    return result